yaksh/templates/yaksh/statistics_question.html

```diff
@@ -0,0 +1,19 @@
+{% extends "manage.html" %}
+
+{% block title %} Statistics {% endblock title %}
+
+{% block css %}
+<link rel="stylesheet" href="{{ URL_ROOT }}/static/yaksh/css/question_quiz.css" type="text/css" />
+{% endblock %}
+{% block subtitle %}
+Statistics for {{quiz_name}}
+{% endblock %}
+{% block manage %}
+<p><b>Total number of participants(including multiple attempts): {{ total }}</b></p>
```

Comment:
The multiple attempts is not clear or very meaningful.  For a given attempt for a given question all I want is the number of people who got it right and the total number of people who took the test.